README.md

```diff
@@ -98,6 +98,13 @@ details.
 **NOTE:** Generators run in RSpec 3.x will now require `rails_helper` instead
 of `spec_helper`.
 
+### `rails_helper` vs `spec_helper`
+As described [in this stackoverflow answer](http://stackoverflow.com/a/24145760/495637), `spec_helper.rb` is for specs which don't depend on Rails (such as specs for classes in the lib directory). `rails_helper.rb` is for specs which do depend on Rails (in a Rails project, most or all of them).
+
+If you want your non-Rails-dependent specs to enforce that they're non-Rails-dependent, and to run as fast as possible when you run them by themselves, you could require `spec_helper.rb` rather than `rails_helper.rb` in those.
```

Comment:
> exists to boot rails for specs that require it

This still strikes me as... unnecessarily difficult. This is `rspec-rails`, right? As in _rspec_... for _Rails_ projects. I mean isn't it true that in 99% of Rails projects, 99% of specs will require Rails? Why not make `.rspec` require `rails_helper` by default?

It seems strange to me that _by default_, the only specs that will run are specs that don't require Rails. If we go to the trouble of generating an `.rspec` file that requires a helper, why not make it require `rails_helper`, so that typical Rails specs will run?

Then if someone wants to do something unusual, they can adjust `.rspec` to only include `spec_helper`